k9mail/src/main/java/com/fsck/k9/mailstore/LocalMessage.java

```diff
@@ -27,18 +27,16 @@
 
 
 public class LocalMessage extends MimeMessage {
-    protected MessageReference mReference;
     private final LocalStore localStore;
 
-    private long mId;
-    private int mAttachmentCount;
-    private String mSubject;
-
-    private String mPreview = "";
-
-    private long mThreadId;
-    private long mRootId;
+    private long databaseId;
```

Comment:
I'd rather write `messageIdHeader` or `messageIdValue` to distinguish the `Message-Id` header from the database ID of a message than having to write `messageDatabaseId` for the database ID of a message. But I don't care too much. However, for consistency reasons, I'd make the same change in `LocalFolder`, i.e. change `id` to `databaseId`.